lib/portlayer/event/collector/vsphere/collector.go

```diff
@@ -176,7 +176,7 @@ func evented(ec *EventCollector, page []types.BaseEvent) {
 			ec.callback(NewVMEvent(page[i]))
 		case *types.VmReconfiguredEvent:
 			// reconfigures happen often, so completely ignore for now
-			return
+			continue
 		default:
```

Comment:
Ugh...nice catch I totally screwed that up...FTR this was added to help identify missed events but this would've made it worse.